### ISSUE TITLE
Wysiwyg html editor plugin for "HTML Editor" textarea in HTML question

### ISSUE THREAD

**Issue by editplus86** [surveyjs/surveyjs#743]
### Are you requesting a feature, reporting a bug or ask a question?
feature/bug

### What is the current behavior?
Wysiwyg html editor not working with HTML Editor tab textarea

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://plnkr.co/edit/9HTarj8PBI0EBuwLnzZQ?p=preview
Add HTML type question and click on Edit, Click on "Html Editor", textarea should be converted to "Wysiwyg html editor"

I am using summernote "Wysiwyg html editor" [ https://summernote.org/ ]

Test code : 

editor.onQuestionAdded.add(function(sender, options){
    var currQuestion = options.question;
    var t = currQuestion.getType();
    console.log(t);
    if (t==='html') {
     $('.modal-body.svd_notopbottompaddings').find('textarea').summernote();
    }
});

### Specify your

* surveyjs platform (angular or react or jquery or knockout or vue): jQuery
* surveyjs version: "0.12.30"

**Comment by andrewtelnov** [comment]
@editplus86 I would suggest to use the latest version 0.12.35, since some bugs have been fixed in Property Editors. However, there is no build-in way to override the property editor yet.
We want to add the ability to override the current properties. For example "text" and "html" with a popular markdown/html editors.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@editplus86 There is a defaultValue property now. You may want to introduce correct answer property with the same type.
In the previous post, I suggested to work together on the quiz functionality. 
We may introduce the correctAnswer. We may even encrypt/descrypt it (something really simple). In this case the correct answer will not be seen on the page.

Thank you,
Andrew

**Comment by editplus86** [comment]
Thanks Andrew :)
Appreciate your prompt reply. 
I would love to work with you infect have created several other type of questions (required for quiz) in a separate plnkr which I will share with you soon for your review and feedback.
I found SurveyJs is not only good fit for Survey functionality but also quiz and online test web application.
Needs few more enhancement in editor to have more flexibility so this can be adopted world wide.
I tried as much as possible in my plnkr.  
 
Let me check the v0.12.35
